### DISCUSSION TITLE
Community update December 2022

### DISCUSSION THREAD

**Discussion by vlidholt** [serverpod/serverpod#497] [Announcements]
Hi Serverpod fans,

We've got some exciting updates to share with you! First, we just released a candidate for Serverpod 1.0, and we'd love for you to try it and let us know what you think. We're also looking for feedback on the documentation - if there's anything you think we missed, let us know, and we will make sure it gets added! Join the discussion here on Github.

We're planning to release the final 1.0 version of Serverpod in early January. We can't thank you enough for your support as we work towards this milestone. 🥳

## Work at Serverpod
In other news, we're on the hunt for a developer experience expert to join our team. This person will be responsible for creating tutorials, videos, and interacting with the Serverpod community. If you or someone you know might be interested in this opportunity, don't hesitate to get in touch. [Read more here](https://careers.serverpod.dev/jobs/6380e1af0b1fbafbd2fb77cf).

## Built with Serverpod
At Serverpod, we are always excited to see what you are building with our framework. We're thrilled to share that Buzz has been using Serverpod to develop their latest product: a menu bar app that integrates with Slack, Github, and other tools we use every day. It gives a quick overview of your team, meetings, and daily tasks. [Try it out here](https://www.joinbuzz.com/devs).

![buzz-git-2](https://user-images.githubusercontent.com/1539812/208311698-c7dabc84-1e7d-480f-a563-030a412f32cd.gif)

## Final words
Finally, we also wanted to give a big shoutout to everyone who has provided feedback and contributed code to Serverpod. Your support is invaluable; we couldn't have gotten this far without you.

For more frequent updates, make sure to follow us on Twitter at [@ServerpodDev](https://twitter.com/ServerpodDev). We couldn't be more excited about the upcoming release and to help you make better backends for Flutter.

Thanks for being part of the Serverpod community!

**Comment by BrutalCoding** [comment]
I am pleased to see you and the team on the hard work that has gone into the development of the upcoming release of Serverpod.

Keep up the great work, and I look forward to using the stable version of Serverpod soon!

**Reply by vlidholt** [reply]
Thank you! 🤩

**Comment by AsherCarneiro** [comment]
yooo, keep up the good work✨
